pkg/test/resourcefixture/testdata/basic/monitoring/v1beta1/monitoringdashboard/monitoringdashboardbasic/_generated_export_monitoringdashboardbasic.golden

```diff
@@ -0,0 +1,45 @@
+apiVersion: monitoring.cnrm.cloud.google.com/v1beta1
+kind: MonitoringDashboard
+metadata:
+  name: monitoringdashboard-${uniqueId}
+spec:
+  columnLayout:
+    columns:
+    - weight: 2
+      widgets:
+      - title: Widget 1
+        xyChart:
+          dataSets:
+          - plotType: LINE
+            timeSeriesQuery:
+              timeSeriesFilter:
+                aggregation:
+                  perSeriesAligner: ALIGN_RATE
+                filter: metric.type="agent.googleapis.com/nginx/connections/accepted_count"
+              unitOverride: "1"
+          timeshiftDuration: 0s
+          yAxis:
+            label: y1Axis
+            scale: LINEAR
+      - text:
+          content: Widget 2
+          format: MARKDOWN
+      - title: Widget 3
+        xyChart:
+          dataSets:
+          - plotType: LINE
+            timeSeriesQuery:
+              timeSeriesFilter:
+                aggregation:
+                  perSeriesAligner: ALIGN_RATE
+                filter: metric.type="agent.googleapis.com/nginx/connections/accepted_count"
+              unitOverride: "1"
+          yAxis:
+            label: y1Axis
+            scale: LINEAR
+      - logsPanel:
+          filter: metric.type="agent.googleapis.com/nginx/connections/accepted_count"
+        title: Widget 4
+  displayName: monitoringdashboard-updated
+  projectRef:
+    external: ${projectId}
```

Comment:
hmmm, do you know why this does not have the `.status`?

Reply:
So I _think_ export should not have status, the intent is that we're going to feed it back into `kubectl apply` (for example).  I could be wrong on that, but I don't think I am.

It wouldn't be hard to support status, particularly once we are generating the KRM <-> proto mapping functions.  But today I don't think we do.

Reply:
Ah! Didn't realize this is *_generated_export_* 
That makes sense.